cfme/tests/infrastructure/test_providers.py

```diff
@@ -214,6 +215,7 @@ def test_api_port_max_character_validation_infra(appliance):
 
 @pytest.mark.rhv1
 @pytest.mark.usefixtures('has_no_infra_providers')
+@pytest.mark.uncollectif(lambda provider: provider.one_of(KubeVirtProvider))
 @pytest.mark.meta(blockers=[BZ(1559796, forced_streams=['5.8', '5.9'],
     unblock=lambda provider: not provider.one_of(RHEVMProvider))])
```

Comment:
Required:
Remove stream: ```5.8```
Also 1559796 is [fixed](https://bugzilla.redhat.com/show_bug.cgi?id=1562788#c7) for 5.9 too. So we can remove this mark.